common/log/tag/values.go

```diff
@@ -79,6 +79,9 @@ var (
 	WorkflowActionExternalWorkflowSignalRequested = workflowAction("add-externalworkflow-signal-requested-event")
 	WorkflowActionExternalWorkflowSignalFailed    = workflowAction("add-externalworkflow-signal-failed-event")
 
+	// upsert workflow search attributes
+	WorkflowActionUpsertWorkflowSearchAttributes = workflowAction("add-upsert-workflow-search-attributes-event")
```

Comment:
according to https://github.com/uber/cadence/blob/0f57f64aeef4d9d04dc44d77074f49849ab7ccea/common/log/tag/values.go#L26-L37

probably `add-workflow-search-attributes-event` ?

Reply:
renamed